index.rst

```diff
@@ -273,7 +260,7 @@ Testimonials
 
   -- Anupam Garg, Datrium, Inc.
 
-Translations
+Contributing
 ------------
 
-* `Chinese <http://postgrest.org/zh/latest/>`_ (latest version ``v0.4.2.0``)
+Please see the `Contributing guidelines <https://github.com/PostgREST/postgrest/blob/main/.github/CONTRIBUTING.md>`_ in the main PostgREST repository.
```

Comment:
Cool. I think we should leave all the dev docs on the main repo.

Btw, I've just noticed that on CONTRIBUTING.md we have links to:

- https://postgrest.org/en/stable/development.html#running-the-test-suite
- https://postgrest.org/en/stable/development.html#build-from-source

I think those can be removed after this PR is done.